### ISSUE TITLE
how to handle missing string files for a build?

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/chipper#313]
Missing string files (sim-specific or common) for a locale currently result in an error.

Sim-specific example:

```
% cd concentration
% grunt --locales=en,es
...
Running "after-requirejs-build" task
Fatal error: Unable to read "/Users/cmalley/PhET/GitHub/babel/concentration/concentration-strings_es.json" file (Error code: ENOENT).
% 
```

Common-code example:

```
% cd beers-law-lab
% grunt --locales=en,es
...
Running "after-requirejs-build" task
Fatal error: Unable to read "/Users/cmalley/PhET/GitHub/babel/scenery-phet/scenery-phet-strings_es.json" file (Error code: ENOENT)
```

These should probably be warnings instead of fatal errors.

**Comment by pixelzoom** [comment]
Converted to a warning in afterRequirejsBuild.js.

Code path was:

`var fileContents = grunt.file.readJSON( stringsFilename );`

New code path:

```js
var fileContents;
try {
  fileContents = grunt.file.readJSON( stringsFilename );
}
catch( error ) {
  grunt.log.warn( 'missing string file: ' + stringsFilename );
  fileContents = {};
}
```

Tested with same scenarios as above:

```
% cd concentration
% grunt --locales=en,es
...
Running "after-requirejs-build" task
>> missing string file: /Users/cmalley/PhET/GitHub/babel/concentration/concentration-strings_es.json

Done, without errors.
% 
```
% cd beers-law-lab
% grunt --locales=en,es
...
Running "after-requirejs-build" task
>> missing string file: /Users/cmalley/PhET/GitHub/babel/scenery-phet/scenery-phet-strings_es.json

Done, without errors.
%
```

Assigning to @jbpet to verify.

**Comment by pixelzoom** [comment]
@jbphet poke.

**Comment by jbphet** [comment]
Looks like the code in question has been moved to getStringMap.js.  I took a look, and it looks good to me. I tested on the arithmetic sim, first using ```grunt --locales=en,es_PE```, which succeeded without errors (as it should), the using ```grunt --locales=en,ak```, which produced the expected warnings.

It should be noted that html files for the specified locales for which no translations exist are still produced, the just use the fallback English strings for everything.  That's cool, but thought it should be noted for anyone who looks at this in the future.

Closing.
